staking_deposit/intl/en/utils/validation.json

```diff
@@ -10,5 +10,19 @@
     },
     "validate_choice": {
         "err_invalid_choice": "That is not one of the valid choices. Please retype your choice."
+    },
+    "validate_eth1_withdrawal_address": {
+        "err_invalid_ECDSA_hex_addr": "The given Eth1 address is not in hexadecimal encoded form.",
+        "err_invalid_ECDSA_hex_addr_checksum": "The given Eth1 address is not in checksum form.",
+        "msg_ECDSA_hex_addr_withdrawal": "**[Warning] you are setting an Eth1 address as your withdrawal address. Please ensure that you have control over this address.**"
+    },
+    "validate_bls_withdrawal_credentials": {
+        "err_not_bls_form": "The given withdrawal credentials is not in BLS_WITHDRAWAL_PREFIX form."
```

Comment:
I assume the error for not bls form will be reverted based on the discussion here https://github.com/ethereum/staking-deposit-cli/pull/315 ?